crypto/ec/ec_backend.c

```diff
@@ -410,6 +410,9 @@ int ossl_ec_key_fromdata(EC_KEY *ec, const OSSL_PARAM params[], int include_priv
 
     param_pub_key =
         OSSL_PARAM_locate_const(params, OSSL_PKEY_PARAM_PUB_KEY);
+    if (param_pub_key == NULL)
+        param_pub_key =
+            OSSL_PARAM_locate_const(params, OSSL_PKEY_PARAM_ENCODED_PUBLIC_KEY);
```

Comment:
It is slightly problematic because `OSSL_PKEY_PARAM_ENCODED_PUBLIC_KEY` does not appear in the importable parameters list:

https://github.com/openssl/openssl/blob/72a85c17aae602e881c917c3f6e93bd7f7260093/providers/implementations/keymgmt/ec_kmgmt.c#L531-L532

But adding it there isn't quite right either - because it will never be exported.

Reply:
Yeah, we would have to make importable and exportable different which would be possible. On the other hand I see this only as a _workaround_ for this clumsiness and I'd be OK with dropping the last commit from the PR altogether.